Base Upload

zliving opened the pull request:
Base Unity project with a few prefabs for characters, cameras, and environments.

zliving committed:
Base Upload

Base Unity project with a few prefabs for characters, cameras, and envoirnments.

zliving committed:
Update Project to 5.5.1f1

cvasqu09 committed:
Added the gitignore file

zliving committed:
Merge pull request #3 from zliving/feature/add-gitignore

Added the gitignore file

cvasqu09 reviewed (changes requested):
Need to place gitignore somewhere else so metadata and cache isn't uploaded.

cvasqu09 committed:
Added a gitignore for the master branch.

We don't want the metadata and everyone's cache to be uploaded when we starting merging with this branch.

zliving committed:
Merge pull request #11 from zliving/cvasqu09-patch-1

Added a gitignore for the master branch.

zliving committed:
Merge branch 'master' into feature/base-unity-project

zliving commented:
Closing this PR since meta data and shader cache isn't needed. Doing another branch to fix this.,
